README-de.md

```diff
@@ -0,0 +1,279 @@
+<summary>📃 Schreibe Artikel und erwähne dein Projekt</summary>
+<p>
+
+> Schreibe Artikel über dein Projekt. Schreib über die technischen Anforderungen, wie dein Projekt funktioniert, Probleme auf die du gestoßen bist, etc. Mögliche Plattformen zum Veröffentlichen:
+>
+> - [medium](https://medium.com/)
+> - [dev.to](https://dev.to/)
+</p>
+</details>
+
+<details>
+<summary>🎤 Präsentiere dein Projekt auf Konferenzen/Treffen</summary>
+<p>
+
+> Wenn du dein Projekt auf Konferenzen oder Treffen präsentierst, steigert das dessen Bekanntheitsgrad.
+
+</p>
+</details>
+
+<details>
+<summary>🎥 Nimm Videos auf, in denen du dein Projekt präsentierst</summary>
+<p>
+
+> Ein Video aufzunehmen ist keine einfache Aufgabe. Allerdings ist es wohl der effizienteste Weg, um dein Projekt bekannt zu machen.
+
+</p>
+</details>
+
+<details>
+<summary>🕐 Wähle die beste Zeit für Social Media Beiträge</summary>
+<p>
+
+> Publiziere nicht während den Ferien oder an Wochenenden. Schreib stattdessen lieber in der Mitte der Woche.
+
+</p>
+</details>
+
+<details>
+<summary>🗑 Spam Plattformen nicht mit deiner Werbung zu</summary>
+<p>
+
+> Publiziere nicht zwei Mal auf der selben Plattform. Das gilt als Spam und könnte schlechtes Licht auf dein Projekt werfen.
+
+</p>
+</details>
+
+<br />
+
+## 3. 🤝 Bleib präsent und in Kontakt
+
+<br />
+
+<details>
+<summary>🆕 Versuche regelmäßig neue Versionen deines Projekts zu veröffentlichen</summary>
+<p>
+
+> Warte und verbessere dein Projekt mit neuen Releases und veröffentliche Changelogs.
+
+</p>
+</details>
+
+<details>
+<summary>❗ Warte dein Projekt, bearbeite offene Tickets</summary>
+<p>
+
+> Lass kein offenes Issue unbeantwortet. Sei nett zu den Leuten, die ein Ticket eröffnet haben. 😉
+
+</p>
+</details>
+
+<details>
+<summary>🙏 Lade Nutzer ein mit zumachen</summary>
+<p>
+
+> Ein gesundes Projekt ist ein Projekt mit einer Community und Mitwirkenden. Lass deine Nutzer wissen, dass du hilfe brauchst, indem du Tickets mit `contribution welcome` oder `good first issue` betitelst. [Siehe github Labels](https://help.github.com/en/articles/about-labels).
+
+</p>
+</details>
+
+<details>
+<summary>🏆 Belohne Mitwirkende</summary>
+<p>
+
+> Sei nett zu Leuten, die dir geholfen haben! Manche Open Source Projekte wie [gatsby](https://github.com/gatsbyjs/gatsby) belohnen Mitwirkende mit Goodies. Wenn du dir das nicht leisten kannst, schreib einen Beitrag (auf Twitter oder anderen Plattformen) und erwähne den Autoren ([hier ist ein Beispiel zu einem öffentlichen Danke](https://twitter.com/FranckAbgrall/status/1139470547492978688)). Eröffne einen `Mitwirkende` Bereich in deiner README, um den Leuten öffentlich zu danken. Du kannst diese Namen auch in deiner Dokumentation oder auf deiner Webseite zeigen. Hier sind ein paar Beispiele:
+>
+> - [vuepress (contributors README section)](https://github.com/vuejs/vuepress#code-contributors)
+> - [Rythm.js (random highlighted contributor on demo page)](https://okazari.github.io/Rythm.js/)
```

Comment:
Translation is missing on the link description :)

Reply:
Thank you very much! I catched up on this